sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -544,7 +544,9 @@ public abstract partial class RequestContent : System.IDisposable
         public static Azure.Core.RequestContent Create(byte[] bytes) { throw null; }
         public static Azure.Core.RequestContent Create(byte[] bytes, int index, int length) { throw null; }
         public static Azure.Core.RequestContent Create(System.IO.Stream stream) { throw null; }
-        public static Azure.Core.RequestContent Create(object serializable, Azure.Core.Serialization.ObjectSerializer? serializer = null) { throw null; }
+        public static Azure.Core.RequestContent Create(object serializable) { throw null; }
+        public static Azure.Core.RequestContent Create(object serializable, Azure.Core.Serialization.ObjectSerializer? serializer) { throw null; }
+        public static Azure.Core.RequestContent Create(object serializable, Azure.Core.Serialization.PropertyNamingConvention propertyNamingConvention) { throw null; }
```

Comment:
So, interestingly, that's exactly what I implemented first [in this PR](https://github.com/Azure/azure-sdk-for-net/pull/36868), but the feedback was that [it was too verbose](https://github.com/Azure/azure-sdk-for-net/pull/36868/files#r1220569893).  Since architects agree that there should only be one overload, I'd like to go with the less-verbose approach and we can introduce the options bag if that makes sense in the future.  If we rename it to ProtocolOptions, there's a chance it would evolve in a way where some of the options wouldn't make sense for RequestContent, and I wouldn't want using it here to limit our options for evolving it.